app/assets/stylesheets/app/user_accounts.scss

```diff
@@ -0,0 +1,15 @@
+td.expired {
+  background-color: $errorBackground !important;
```

Comment:
why do we need the `!important` here?

Reply:
@giladshanan there's a selector, `.table-striped tbody > tr:nth-child(odd) > td`, that alternates the color that's pretty specific. So `!important` is used to override it.